resources/puppetlabs/lein-ezbake/template/foss/ext/build_defaults.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 default_cow: 'base-bionic-i386.cow'
-cows: 'base-bionic-i386.cow base-buster-i386.cow base-focal-i386.cow base-bullseye-i386.cow base-jammy-i386.cow'
+cows: 'base-bionic-i386.cow base-buster-i386.cow base-focal-i386.cow base-bullseye-i386.cow base-jammy-i386.cow base-bookworm-i386.cow'
```

Comment:
Is this required?  As far as I can see, most packages are architecture agnostic with the exception of the `pdk`, `puppet-agent` and `puppet-bolt` ones, and for these no i386 packages are available for the oldstable release (both for Puppet 7 and Puppet 8) so I don't expect you will build some for the stable release?

https://apt.puppet.com/pool/bullseye/puppet7/p/pdk/index.html
https://apt.puppet.com/pool/bullseye/puppet7/p/puppet-agent/index.html
https://apt.puppet.com/pool/bullseye/puppet7/p/puppet-bolt/index.html
https://apt.puppet.com/pool/bullseye/puppet8/p/pdk/index.html
https://apt.puppet.com/pool/bullseye/puppet8/p/puppet-agent/index.html
https://apt.puppet.com/pool/bullseye/puppet8/p/puppet-bolt/index.html

Reply:
Legacy stuff. It should be cleaned up at some point. build.rake parses the cows, and likely a bunch of puppet internal pipelines pass it around.

Reply:
Fine, thanks for the clarification!